src/super_gradients/training/datasets/dataset_interfaces/dataset_interface.py

```diff
@@ -782,9 +782,12 @@ class CoCoDetectionDatasetInterface(DetectionDatasetInterface):
     def __init__(self, dataset_params={}):
         super(CoCoDetectionDatasetInterface, self).__init__(dataset_params=dataset_params)
 
-        train_input_dim = (self.dataset_params.train_image_size, self.dataset_params.train_image_size)
         targets_format = get_param(self.dataset_params, "targets_format", DetectionTargetsFormat.LABEL_CXCYWH)
 
+        train_input_dim = (self.dataset_params.train_image_size, self.dataset_params.train_image_size)
+        train_tight_box_rotation = get_param(self.dataset_params, "train_tight_box_rotation", False)
```

Comment:
As in previous PRs- We are planning to take all of the defaults out of dataset interface code to be configured only through the YAML files (so add them in the right dataset_params in recipes).

Reply:
changed in f1dc9d6f317f30c415d7c26cb912c4ebc2287307